package.json

```diff
@@ -20,7 +22,8 @@
     "preversion": "echo BE SURE TO UPDATE THE VERSION NUMBER IN SW.JS | chalk red bold",
     "version:patch": "npm --silent version patch",
     "version:minor": "npm --silent version minor",
-    "version:major": "npm --silent version major"
+    "version:major": "npm --silent version major",
+    "purgecss": "./scripts/purgecss.js"
```

Comment:
Should we add something to the README about this or does it run auto-magically?

Reply:
I updated the readme, now to start this you do `npm start` (it will start jekyll automatically), and to run a build do `npm run build`